File: populate.js
const request = require('request');

function randomRange(min, max) {
    return Math.random() * (max - min) + min;
}

for (let i = 0; i < 100; i++) {
    const start = new Date(2020, 0).getTime();
    const end = new Date(2021, 0).getTime();
    const timestamp = new Date(randomRange(start, end)).getTime();
    const req = request.post('http://localhost:3000/api/readings', {
        headers: {
            'Authorization': 'Api-Key 1234',
            'Content-Type': 'application/json',
        },
        body: JSON.stringify({
            sensorId: '5f41a70283d15aa51cec71b7',
            timestamp,
            temperature: randomRange(20, 40),
            pressure: randomRange(10, 30),
            humidity: randomRange(0, 100),
        }),
    });
    req.on('response', res => {

    });
}

File: populate.py
from datetime import datetime

import requests
import random

while True:
    start = 0
    end = int(datetime.now().timestamp())
    timestamp = random.randrange(start, end)
    time = datetime.fromtimestamp(timestamp).isoformat()
    response = requests.post(
        'http://localhost:3000/api/readings',
        headers={
            'Authorization': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOjEsInJvbGVzIjpbInN0YXRpb24iXSwiaWF0IjoxNjI4MDU2NjE2fQ.3hIlQ7HPQP6Th8SP-SYkqmOHQkInMm2BUSwLLzCpUUc',
            'Content-Type': 'application/json',
        },
        json={
            'time': time,
            'temperature': random.randrange(0, 100),
            'rain1h': random.randrange(0, 10),
        })
    print(response.status_code)
